project/events/views.py

```diff
@@ -3,7 +3,13 @@
 from django.views.generic.list import ListView
 
 from .models import Event
+from django.db.models.functions import Now
 
 
 class EventListView(ListView):
     model = Event
+    queryset = Event.objects.filter(start_time__gte=Now()).order_by('start_time')
+
+class PastEventListView(EventListView):
```

Comment:
No need to inherit from `EventListView`, just inherit from `ListView` and specify `model = Event` again.